main.go

```diff
@@ -10,7 +11,7 @@ import (
 )
 
 func main() {
-	e := echo.New()
+	e := middleware.Create()
```

Comment:
squibble: this maybe looks a little weird. Like it's creating middleware, not the server object with middleware added to it. It might be worth changing the signature to be more like `addRouteHandlers` and modify an existing echo object.